shared/ui/core/src/main/java/com/ivy/ui/FormatMoneyUseCase.kt

```diff
@@ -23,25 +32,35 @@ class FormatMoneyUseCase @Inject constructor(
     private val withoutDecimalFormatter = DecimalFormat("###,###", DecimalFormatSymbols(locale))
     private val withDecimalFormatter = DecimalFormat("###,###.00", DecimalFormatSymbols(locale))
     private val shortenAmountFormatter = DecimalFormat("###,###.##", DecimalFormatSymbols(locale))
+    private val cryptoFormatter = DecimalFormat("#".repeat(9), DecimalFormatSymbols(locale))
```

Comment:
Ah, I mean we need to support showing any arbitrary crypto with up to 9 decimals precision without showing unnecessary trailing digits. The correct pattern should be:
- `"###,###." + "#".repeat(9)`

Reply:
Done